lib/MusicBrainz/Server/WebService/Serializer/JSON/2/Release.pm

```diff
@@ -82,8 +81,14 @@ sub serialize
     $body{collections} = list_of($entity, $inc, $stash, "collections")
         if $inc && ($inc->collections || $inc->user_collections);
 
-    $body{"release-group"} = serialize_entity($entity->release_group, $inc, $stash)
-        if $inc && $inc->release_groups;
+    if ($inc && $inc->release_groups)
+    {
+        # MBS-9129: If ratings are requested (even though a release doesn't have any), those of the release group should be
```

Comment:
I'm used to 132 as 80 can get very restrictive (not so much for perl, but for C++ it is), so my emacs is set up for that.
I can reformat those lines, no problem.